stratum/hal/lib/common/common.proto

```diff
@@ -276,6 +276,12 @@ message PortConfigParams {
     DEVICE_TYPE_VIRTIO_NET = 1;
     DEVICE_TYPE_VIRTIO_BLK = 2;
   }
+  // Packet Direction types
+  enum PacketDirection {
+    DIRECTION_HOST = 0;
+    DIRECTION_NETWORK = 1;
+    DIRECTION_NONE = 2;
```

Comment:
Do we care about having a `DIRECTION_NONE`, if we need it for default value, then we should put it a 1st element in the enum.

Reply:
`DIRECTION_NONE` can be useful for PSA. The order of fields corresponds to PNA values (RX=0, TX=1), but I'm fine with changing it.

Reply:
OK, then let's keep it as is.

Reply:
With the conditional assignment in place already, changing the order was really easy.